tests/main.c

```diff
@@ -21,9 +21,7 @@ int main(int argc, char *argv[])
 	cl_sandbox_set_search_path_defaults();
 
 	/* Run the test suite */
-	res = clar_test_run();
-
-	clar_test_shutdown();
+	res = clar_test();
```

Comment:
This seems like a change which isn't really related to your actual topic. If you think this should be done regardless, please put it in a separate commit and explain why we should use that way instead.